transformers/test_data/config_values.go

```diff
@@ -40,6 +40,26 @@ func FlipAddresses() []string {
 	return addressesResult
 }
 
+func Flip110Addresses() []string {
+	var addressesResult []string
+	flipAddresses := constants.GetContractAddresses([]string{
+		"MCD_FLIP_BAT_A_1.1.0",
+		"MCD_FLIP_ETH_A_1.1.0",
+		"MCD_FLIP_KNC_A_1.1.0",
+		"MCD_FLIP_MANA_A_1.1.0",
+		"MCD_FLIP_TUSD_A_1.1.0",
+		"MCD_FLIP_USDC_A_1.1.0",
+		"MCD_FLIP_USDC_B_1.1.0",
+		"MCD_FLIP_WBTC_A_1.1.0",
+		"MCD_FLIP_ZRX_A_1.1.0",
```

Comment:
I think we can probably shore this up after the backfill process is running, but in theory this should include the PAXUSD + USDT addresses

Reply:
ugg.. forgot about this file.